projects/ngx-cookiebot/package.json

```diff
@@ -30,7 +30,7 @@
     "url": "https://github.com/halloverden/ngx-cookiebot/tree/main/projects/ngx-cookiebot"
   },
   "peerDependencies": {
-    "@angular/common": "8.2.14 - 15",
-    "@angular/core": "8.2.14 - 15"
+    "@angular/common": "9 - 16",
+    "@angular/core": "9 - 16"
```

Comment:
If you're not able to test each version from 9-16, I suggest you change both these to  ^16.0.0, and we'll see later if we want to extend version support or release a new major when A17 comes around

Reply:
Fixed it after merging